wiki/Tournaments/RMoT_Invitational/1/en.md

```diff
@@ -0,0 +1,115 @@
+# RMoT Invitational 1
+
+The Rapid Monthly osu! Tournament Invitational \#1 is a tournament organized and ran by the Rapid Tournaments team.
+It is an exclusive tournament where only players who reached semi-finals in one of the last 4 RMoT's get invited.
+
+The tournament brackets can be found [on Challonge.](https://challonge.com/rmotinvitational1). Most of the information here complies with the [forum post](https://osu.ppy.sh/forum/t/538852).
+
+
+![][flag_US] [\[Sony\]](https://osu.ppy.sh/u/8750694) (formerly known as sonyguy699) won the Invitational, beating ![][flag_PL] [CarlosH20](https://osu.ppy.sh/u/8102219) in the finals. 
+
+## Schedule
+
+The RMoT Invitational are held over two weekends.
+
+| Event                               | Date        |
+|------------------------------------:|:------------|
+| Round of 16                         | 7 Jan 2017  |
```

Comment:
full stops after the day and prefixed zero
`07. Jan 2017`
_(suppressing other instances)_